<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#okButton {
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color:rgb(44, 47, 51);
    font: bold 25px;
    padding: 1px;
}
QPushButton#okButton:pressed {
    background-color: rgb(88, 101, 242,);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>✅</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#cancelButton {
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color:rgb(44, 47, 51);
    font: bold 25px;
    padding: 1px;
}
QPushButton#cancelButton:pressed {
    background-color: rgb(88, 101, 242,);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>❌</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="taskname">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: bold 20px;</string>
      </property>
      <property name="text">
       <string>Task name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="info_inp">
      <property name="styleSheet">
       <string notr="true">background-color: white;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color:rgb(44, 47, 51);
font: bold 14px;
padding: 1px;
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timelabel">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
 font: bold 20px;</string>
      </property>
      <property name="text">
       <string>Time: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="timePeeker">
      <property name="styleSheet">
       <string notr="true">background-color: white;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color:rgb(44, 47, 51);
font: bold 14px;
padding: 1px;</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="high_peeker">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: bold 20px;
border-width: 2px;
border-radius: 15px;
background-color: rgb(46, 48, 66);</string>
   </property>
   <property name="text">
    <string>Adding a new task:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(88, 101, 242);
border-style: outset;
border-width: 3px;
border-radius: 20px;
border-color: rgb(46, 48, 66);
font: bold 14px;
padding: 1px;

</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>okButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>high_peeker</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
